src/molecules/ClusterResult.jsx

```diff
@@ -0,0 +1,126 @@
+import React, { useState, useEffect, useRef } from "react";
+import styled from "styled-components";
+import Button from "../atoms/Button";
+import resultBtnImage from "../images/results_button.svg";
+import chevron_lite from "../images/chevrons_up_lite.svg";
+import chevron_dark from "../images/chevrons_up_dark.svg";
+
+import { useHistory } from "react-router-dom";
+
+const Container = styled.div`
+    max-width= "575px";
+    width: ${window.innerWidth > 575 ? "567px" : window.innerWidth};
+    position:absolute;
+    bottom: 0;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+`;
+
+const ResultsBox = styled.div`
+  max-width: 575px;
+  width: 98vw;
+  background-color: ${({ theme }) => theme.body};
+  color: ${({ theme }) => theme.text};
+  display: flex;
+  height: ${({ expanded }) => (expanded ? "500px" : "140px")};
+  flex-direction: column;
+  justify-content: top;
+  align-items: center;
+  border-radius: 15px 15px 0px 0px;
+  transition: height 2s;
+`;
+
+const ResultHandleImg = styled.img`
+  margin: 10px;
+`;
+
+const ChevronsImg = styled.img`
+  margin: 10px;
+`;
+
+const ResultBoxText = styled.b`
+  color: ${({ theme }) => theme.text};
+  margin-top: 3vh;
+`;
+
+const ResultName = styled.b`
+  color: ${({ theme }) => theme.text};
+  font-size: 5vh;
+  margin-top: 2vh;
+`;
+
+const resultText = loadingState => {
+  switch (loadingState) {
+    case "loading":
+      return "Loading results...";
+    case "none":
+      return "No results found";
+    case "found":
+      return "Is this an";
+    default:
+      return "Error, please supply a result";
+  }
+};
+
+const getLoadingState = ClusterData => {
```

Comment:
try and refactor from a nested conditional to if,elseif, if and see it codeclimate reduces the cognitive complexity score